src/hooks/useTxStatus/useTxStatus.ts

```diff
@@ -0,0 +1,71 @@
+import { usePrevious } from '@chakra-ui/react'
+import type { AccountId } from '@shapeshiftoss/caip'
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useEffect, useMemo } from 'react'
+import { selectTxById } from 'state/slices/selectors'
+import { serializeTxIndex } from 'state/slices/txHistorySlice/utils'
+import { useAppSelector } from 'state/store'
+
+type UseTxStatusProps = {
+  accountId?: AccountId | null
+  txId?: string | null
```

Comment:
suggestion: following recent discussions, this may disambiguate things

```suggestion
  txHash?: string | null
```

Reply:
fixed in [77b81a5](https://github.com/shapeshift/web/pull/7043/commits/77b81a593dc0a6c7088883c3ee9d54c59b297636)

Reply:
f5be8cc3e34df8a055d6bcbc0dee11421dc22802